Python/bltinmodule.c

```diff
@@ -540,6 +540,36 @@ filter_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
     return (PyObject *)lz;
 }
 
+static PyObject *
+filter_vectorcall(PyObject *type, PyObject * const*args,
+                size_t nargsf, PyObject *kwnames)
+{
+    assert(PyType_Check(type));
+    if (!_PyArg_NoKwnames("filter", kwnames)) {
```

Comment:
filter_new() only checks if `type == &PyFilter_Type` is true. Why is it different for vectorcall?